great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -64,14 +63,14 @@ def __init__(
         sampling_method: Optional[str] = "bootstrap",
         enforce_numeric_metric: Optional[Union[str, bool]] = True,
         replace_nan_with_zero: Optional[Union[str, bool]] = True,
-        confidence_level: Optional[Union[float, str]] = 9.5e-1,
-        num_bootstrap_samples: Optional[Union[int, str]] = None,
-        round_decimals: Optional[Union[int, str]] = None,
+        confidence_level: Optional[Union[str, float]] = 9.5e-1,
```

Comment:
rename to "false_positive_rate"; should be the desired false positive rate. We're no longer exposing confidence interval in this implementation (though we could estimates of parameter quality in meta (in a future PR))

Reply:
Done!